dali/test/python/test_pipeline_debug.py

```diff
@@ -187,12 +187,13 @@ def es_pipeline_standard():
 
 def test_external_source_debug_sample_pipeline():
     n_iters = 10
+    prefetch_queue_depth = 2
     pipe_load = load_images_pipeline()
-    pipe_standard = es_pipeline_standard()
-    pipe_debug = es_pipeline_debug()
+    pipe_standard = es_pipeline_standard(prefetch_queue_depth=prefetch_queue_depth)
+    pipe_debug = es_pipeline_debug(prefetch_queue_depth=prefetch_queue_depth)
     pipe_load.build()
     pipe_debug.build()
-    for _ in range(n_iters):
+    for _ in range(n_iters + prefetch_queue_depth):
```

Comment:
I would add a comment why we need that `+ prefetch_queue_depth` here.

Reply:
done